### DISCUSSION TITLE
Is there any parameter for chosing length of the responses?

### DISCUSSION THREAD

**Discussion by SuSl1KK** [dsdanielpark/Bard-API#256] [Q&A]
Is there any way to change length of the Bard's responses? On the Bard website you can adjust lengths of the responses, so im curious if I can do this using this library

**Comment by dsdanielpark** [comment]
As stated in the readme and FAQ, the parameters that can be adjusted on Bard's website vary significantly by country, region, and account. (For example, image uploading was very late to be available in Korea.) While it is possible to arbitrarily adjust the length of responses, this is merely a method of slicing answers and may not be very useful. (Reference: https://github.com/dsdanielpark/Bard-API/blob/main/documents/README_DEV.md#max_token-max_sentence)

In conclusion, limiting the length of responses by adjusting model parameters is currently difficult to support at the level of this Python package.
